api/src/main/resources/messages.properties

```diff
@@ -631,6 +631,8 @@ reporting.parameter.startDate=Start Date
 reporting.parameter.endDate=End Date
 reporting.parameter.location=Location
 reporting.parameter.locationList=Locations
+reporting.parameter.attributeType=Attribute Type
+reporting.parameter.values=Value
```

Comment:
I mean that since the text is "Value" shouldn't the key be "reporting.parameter.value" without the s at the end?